app/models/like.rb

```diff
@@ -0,0 +1,8 @@
+class Like < ApplicationRecord
+  belongs_to :user
```

Comment:
- [ ] @AlaaAlsalem as previously suggested kindly also add the cases associated with the foreign key.

Reply:
Well resolved 👍🏽